applications/luci-app-dockerman/Makefile

```diff
@@ -0,0 +1,12 @@
+include $(TOPDIR)/rules.mk
+
+PKG_LICENSE:=AGPL-3.0
+PKG_MAINTAINER:=lisaac <https://github.com/lisaac/luci-app-dockerman>
```

Comment:
Maintainer should be normal name & email  (so that e.g. messaging to maintainers can work).